homeassistant/components/insteon/__init__.py

```diff
@@ -66,6 +68,36 @@
 EVENT_BUTTON_OFF = 'insteon.button_off'
 EVENT_CONF_BUTTON = 'button'
 
+
+# Adapted from:
+# https://github.com/alecthomas/voluptuous/issues/115#issuecomment-144464666
+def set_default_port() -> Callable:
```

Comment:
There's no need to have this nesting, since we don't need to pass in any other arguments than the dict we want to validate.